tests/security/apparmor/aa_status.pm

```diff
@@ -0,0 +1,46 @@
+# Copyright (C) 2018 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# Summary: Test the basic information output function for apparmor aa-status
+# Maintainer: Wes <whdu@suse.com>
+# Tags: poo#36874, tc#1621138
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+
+sub run {
+    select_console 'root-console';
+
+    systemctl('restart apparmor');
+
+    validate_script_output "aa-status", sub {
+        m/
+        module\ is\ loaded.*
+        profiles\ are\ loaded.*
+        profiles\ are\ in\ enforce\ mode.*
+        profiles\ are\ in\ complain\ mode.*
+        processes\ are\ in\ enforce\ mode.*
+        processes\ are\ in\ complain\ mode.*
+        processes\ are\ unconfined/sxx
+    };
+}
+
+sub test_flags {
+    return {milestone => 1, fatal => 1};
```

Comment:
Why do you need these flags? I don't think the tests need to immediately abort if this module fails. This also applies to the other file

Reply:
I have removed the flags. Thank you for reminding.